lumicks/pylake/tests/test_channels/test_channels.py

```diff
@@ -734,11 +748,16 @@ def test_downsampling_like():
     ):
         s.downsampled_like(s)
 
-    for offset in (-4 * 4, t_downsampled[-1] + 1):
+    timestep = 4
+    for offset in (with_offset(-4 * timestep), t_downsampled[-1] - timestep + 1):
         with pytest.raises(RuntimeError, match="No overlap between slices"):
-            s = channel.Slice(channel.Continuous(np.array([1, 2, 3, 4]), with_offset(offset), 4))
+            s = channel.Slice(channel.Continuous(np.array([1, 2, 3, 4]), offset, timestep))
             s.downsampled_like(reference)
 
+    for offset in (with_offset(-4 * timestep + 1), t_downsampled[-1] - timestep):
+        s = channel.Slice(channel.Continuous(np.array([1, 2, 3, 4]), offset, timestep))
+        s.downsampled_like(reference)
```

Comment:
should we actually test that the downsampled result is valid and not just that it doesn't throw? or is that tested enough elsewhere?

Reply:
It is, but not in this particular context, so I've added a quick assert with the result here now.